client/src/contexts/NotificationContext.js

```diff
@@ -0,0 +1,38 @@
+import React, { createContext, useState } from 'react'
+
+export const NotificationContext = createContext({})
+
+export const NotificationContextProvider = ({ children }) => {
+  const [notifications, setNotifications] = useState([
+    {
+      test: {
+        id: 'test',
+        type: 'success',
+        message: 'this is fun!'
+      }
+    }
+  ])
```

Comment:
Since we removed the nested object, we can also clean up the outer dynamic key (e.g., 'test') from the mock data:

```suggestion
  const [notifications, setNotifications] = useState([
    {
        id: 'test',
        type: 'success',
        message: 'this is fun!'
    }
  ])
```